growforest/growforest.go

```diff
@@ -926,40 +927,37 @@ func main() {
 		fmt.Printf("Error: %v\n", bb.TallyError(testtarget))
 
 		if testtarget.NCats() != 0 {
-			falsesbypred := make([]int, testtarget.NCats())
-			predtotals := make([]int, testtarget.NCats())
+			falselyLabelledAs := make([]int, testtarget.NCats())
+			labelledAs := make([]int, testtarget.NCats())
 
-			truebytrue := make([]int, testtarget.NCats())
-			truetotals := make([]int, testtarget.NCats())
+			correctlyLabelledAs := make([]int, testtarget.NCats())
+			sampleCount := make([]int, testtarget.NCats())
 
 			correct := 0
 			nas := 0
 			length := testtarget.Length()
 			for i := 0; i < length; i++ {
-				truei := testtarget.(*CloudForest.DenseCatFeature).Geti(i)
-				truetotals[truei]++
+				trueLabel := testtarget.(*CloudForest.DenseCatFeature).Geti(i)
+				sampleCount[trueLabel]++
 				pred := bb.Tally(i)
 				if pred == "NA" {
 					nas++
 				} else {
-					predi := testtarget.(*CloudForest.DenseCatFeature).CatToNum(pred)
-					predtotals[predi]++
+					predictedLabel := testtarget.(*CloudForest.DenseCatFeature).CatToNum(pred)
+					labelledAs[predictedLabel]++
 					if pred == testtarget.GetStr(i) {
 						correct++
-						truebytrue[truei]++
+						correctlyLabelledAs[trueLabel]++
 					} else {
-
-						falsesbypred[predi]++
+						falselyLabelledAs[predictedLabel]++
 					}
 				}
 
 			}
-			fmt.Printf("Classified: %v / %v = %v\n", correct, length, float64(correct)/float64(length))
+			log.Printf("Correctly classified: %d / %d = %.4f\n", correct, length, float64(correct)/float64(length))
```

Comment:
No reason for this file in particular, it's rather a habit, `fmt.Printf()` is definitely more consistent here. Feel free to make the change!